src/dashboard/src/components/administration/views.py

```diff
@@ -456,11 +457,14 @@ def general(request):
     dashboard_uuid = helpers.get_setting('dashboard_uuid')
     try:
         pipeline = storage_service._get_pipeline(dashboard_uuid)
-    except Exception:
-        messages.warning(request, _("Storage server inaccessible. Please contact an administrator or update storage service URL below."))
-    else:
-        if not pipeline:
-            messages.warning(request, _("This pipeline is not registered with the storage service or has been disabled in the storage service. Please contact an administrator."))
+    except Exception, err:
+        if err.response is not None and err.response.status_code == 404:
+            if storage_form.is_valid():
+                setup_pipeline_in_ss(storage_form.cleaned_data['storage_service_use_default_config'])
```

Comment:
Should there be some exception handling here? I noticed that if I simply disable the pipeline in the SS and then try to submit this form I'm getting an uncaught `HTTPError/requests.exceptions.RequestException` because the SS is returning 500 when you try to create a pipeline that already exists: `{"error_message": "(1062, \"Duplicate entry ...`.